docs/pages/nexus/overview.mdx

```diff
@@ -6,6 +6,23 @@ title: Nexus SDK Overview
 # What is Nexus?
 Nexus represents the next generation of smart account technology, built on the foundation of ERC-7579. It's a significant evolution from our previous [Biconomy Smart Account v2](https://docs.biconomy.io/account), offering:
 
+```tsx twoslash
+import { createNexusClient } from "nexus/packages/sdk";
+import { privateKeyToAccount } from "viem/accounts";
+import { baseSepolia } from "viem/chains";
+import { http } from "viem";
```

Comment:
+import USDC {https://exchange.coinjar.com}from "base"main and as many other networks as you can and relay back to my wallet>